kubernetes/pom.xml

```diff
@@ -17,12 +17,17 @@
     <build>
         <resources>
             <resource>
+                <directory>src/main/catalog</directory>
+                <filtering>false</filtering>
+            </resource>
+            <resource>
                 <directory>src/main/resources</directory>
                 <filtering>false</filtering>
             </resource>
             <resource>
                 <directory>src/main/examples</directory>
                 <filtering>false</filtering>
+                <targetPath>examples</targetPath>
```

Comment:
Unsure about packaging examples ... 
I think its useful for documentation but would you ever deploy the examples as is?